### PR TITLE
Automated reporting

### PR DESCRIPTION
**Pull request by tmcadam**

### CODE REVIEW

**Review by pindge — DISMISSED** [review]

**Comment by tmcadam** [comment]
@pindge I think is ready to merge to master. Could you please review again.

File: dags/infra/variables.py [hunk 1]
```diff
@@ -42,6 +42,9 @@
...
 
 SENTINEL_2_ARD_INDEXING_AWS_USER_SECRET = Variable.get(
-     "sentinel_2_ard_indexing_aws_user_secret", 
-     "sentinel-2-ard-indexing-creds"
- )
\ No newline at end of file
+    "sentinel_2_ard_indexing_aws_user_secret", "sentinel-2-ard-indexing-creds"
+)
+
+S2_NRT_AWS_CREDS = "wagl-nrt-aws-creds"
```
Comment (pindge) [inline dags/infra/variables.py:48]: Is this related to your dags? IT looks likes it related to WAGL

File: dags/infra/connections.py [hunk 2]
```diff
@@ -2,7 +2,12 @@
 # connection ids created by terraform
 """
 
+# Automated reporting connections
 DB_ODC_READER_CONN = "db_odc_reader"  # qa
+DB_REP_WRITER_CONN_DEV = "db_rep_writer_dev"
```
Comment (pindge) [inline dags/infra/connections.py:7]: is this created in Terraform? if not, do not put it here

**Review by pindge — CHANGES_REQUESTED** [review]

File: dags/infra/connections.py (hunk 2 above)
Comment (tmcadam) [reply dags/infra/connections.py:7]: No. Can it be added to Terraform?

Comment (pindge) [reply dags/infra/connections.py:7]: did you create this via airflow connection UI?

File: dags/infra/variables.py (hunk 1 above)
Comment (tmcadam) [reply dags/infra/variables.py]: No , not related to my dags. I'll take it out, might have got this by mistake when merging commits from develop.

File: dags/infra/connections.py (hunk 2 above)
Comment (tmcadam) [reply dags/infra/connections.py:7]: Yes, created it in sandbox UI

Comment (pindge) [reply dags/infra/connections.py:7]: Happy for you to create a `connections.py` within your project, but this isnt the right place for it.

Comment (tmcadam) [reply dags/infra/connections.py:7]: Okay, I'll move  these into our folder?
```
DB_REP_WRITER_CONN_DEV = "db_rep_writer_dev"
DB_REP_WRITER_CONN_PROD = "db_rep_writer_prod"
S3_REP_CONN = "s3_bucket_rep"
```
and the variable 
`COP_API_REP_CREDS = "copernicus_api_password"
`

Comment (pindge) [reply dags/infra/connections.py:7]: yes

**Review by pindge — APPROVED** [review]

**Comment by pindge** [comment]
LGTM

**Pull request merged**